api/v1alpha1/condition_consts.go

```diff
@@ -52,10 +52,7 @@ const (
 
 // Conditions related to the VirtualMachineImages
 const(
-	// VirtualMachineImageOSTypeSupportedCondition denotes that the OS type in the VirtualMachineImage object is
-	// supported by VMService. A VirtualMachineImageOsTypeSupportedCondition is marked true:
-	// - If OS Type is of Linux Family
-	// - If OS Type is supported by hosts in the cluster
```

Comment:
Is there a reason we are removing these comments? Just adding the `//Deprecated` should be enough?

Reply:
yeah. you're probably right.

Reply:
My rationale was why try to describe something that is deprecated. I also remember doing something similar to a few status fields in the VirtualMachineImage.

Reply:
I think we can leave the comments. Would hurt the readability to have a field without a description.